src/erlcloud_ddb2.erl

```diff
@@ -1875,7 +1875,8 @@ dynamize_sse_specification({enabled, Enabled}) when is_boolean(Enabled) ->
                             {local_secondary_indexes, local_secondary_indexes()} |
                             {global_secondary_indexes, global_secondary_indexes()} |
                             {sse_specification, sse_specification()} |
-                            {stream_specification, stream_specification()}.
+                            {stream_specification, stream_specification()} |
+                            {provisioned_throughput, {read_units(), write_units()}}.
```

Comment:
picky, but can you put this between `global_secondary_indexes` and `sse_specification`? I think these are ordered lexically—or intended to be.

Reply:
Sure. There's another occurrence in that file that is not at all ordered alphabetically, though.

Reply:
09bbab1